subiquity/core.py

```diff
@@ -30,6 +31,18 @@
 log = logging.getLogger('subiquity.core')
 
 
+DEBUG_SHELL_INTRO = _("""\
+Installer shell session activated.
+
+This shell session is running inside the installer environment.  You
+will be returned to the installer when this shell is exited, for
+example by typing Control-D or 'exit'.
+
+Be aware that this is an ephemeral environment.  Changes to this
+environment will not survive a reboot. If the install has started, the
+installed system will be mounted at /target.""")
+
+
```

Comment:
Can we do a confirm here? Ie.

```
-installed system will be mounted at /target.""")
+installed system will be mounted at /target.
+
+Press Enter to start shell...""")
```

Reply:
Why? Won't people have already hit some key to get to it? What they really need is just to acknowledge that popup.